contrib/examples/walkthrough/ups-servicebroker.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBroker
+metadata:
+  name: ups-broker
```

Comment:
If you don't specify the namespace attribute here, it will use the default namespace unless you specify one on the command line for kubectl create.   Might be worth a comment here in here, WDYT?